lua/auto-dark-mode/init.lua

```diff
@@ -73,42 +77,46 @@ local function init()
 	elseif system == "Linux" then
 		if not vim.fn.executable("dbus-send") then
 			error([[
-        `dbus-send` is not available. The Linux implementation of
-        auto-dark-mode.nvim relies on `dbus-send` being on the `$PATH`.
-      ]])
+		`dbus-send` is not available. The Linux implementation of
+		auto-dark-mode.nvim relies on `dbus-send` being on the `$PATH`.
+	  ]])
 		end
 
 		query_command = table.concat({
 			"dbus-send --session --print-reply=literal --reply-timeout=1000",
-			"--dest=org.freedesktop.portal.Desktop",
-			"/org/freedesktop/portal/desktop",
-			"org.freedesktop.portal.Settings.Read",
-			"string:'org.freedesktop.appearance'",
-			"string:'color-scheme'",
-		}, " ")
+			"--dest=org.freedesktop.portal.Desktop", "/org/freedesktop/portal/desktop",
+			"org.freedesktop.portal.Settings.Read", "string:'org.freedesktop.appearance'",
+			"string:'color-scheme'"
+			}, " ")
+	elseif system == "Windows_NT" then
+		-- Don't swap the quotes; it breaks the code
+		query_command =
+			'Reg Query "HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize" /v AppsUseLightTheme | findstr "AppsUseLightTheme"'
 	else
 		return
 	end
 
-	if vim.loop.getuid() == 0 then
-		query_command = "su - $SUDO_USER -c " .. query_command
+	if system == "Linux" or system == "Darwin" then
```

Comment:
This could be
```suggestion
	if vim.fn.has("unix") then
```

Reply:
Oh great! This is definitely better!

Reply:
hi, winston! could you please check your matrix dms? :)

Reply:
`if vim.fn.has("unix") then` should be `if vim.fn.has("unix") ~= 0 then` because the function `vim.fn.has("unix")` returns 0, but it doesn't evaluate to false for some reason. This broke the plugin, but after making the change about, it works again.

Reply:
thanks! fixed!

Reply:
@adityamwagh good catch, I always forget that 0 doesn't behave as false in Lua.

Reply:
Yeah, https://www.lua.org/pil/2.2.html says 0 does not mean false! So strange 😆!